cmd/block-generator/runner/run.go

```diff
@@ -125,8 +162,9 @@ func recordDataToFile(start time.Time, entry Entry, prefix string, out *os.File)
 	record("_cumulative", metrics.PostgresEvalName, floatTotal)
 	record("", metrics.ImportedRoundGaugeName, intTotal)
 
-	if writeErrors.Len() > 0 {
-		return fmt.Errorf("error writing metrics (%s): %w", writeErrors.String(), writeErr)
+	// Allow missing eval time metric
```

Comment:
why can it be missing?

Reply:
I added the exception here because this metric isn't in the build I was using

Reply:
Can we make an exception for that metric specifically. Otherwise, if there is another missing metric instead, we will not return an error.

Reply:
I'll remove it, this exception doesn't need to be checked in